sources/packages/backend/apps/queue-consumers/src/processors/schedulers/cra-integration/models/process-response.dto.ts

```diff
@@ -1,4 +1,4 @@
-export class ProcessResponseQueueOutDTO {
+export class ProcessResponseQueue {
```

Comment:
I believe that this is no longer needed and we can use the new summary interface.